ambari-server/src/main/python/ambari-server.py

```diff
@@ -555,6 +555,8 @@ def init_ldap_setup_parser_options(parser):
   parser.add_option('--ldap-sync-username-collisions-behavior', default=None, help="Handling behavior for username collisions [convert/skip] for LDAP sync", dest="ldap_sync_username_collisions_behavior")
   parser.add_option('--ldap-force-lowercase-usernames', default=None, help="Declares whether to force the ldap user name to be lowercase or leave as-is", dest="ldap_force_lowercase_usernames")
   parser.add_option('--ldap-pagination-enabled', default=None, help="Determines whether results from LDAP are paginated when requested", dest="ldap_pagination_enabled")
+  parser.add_option('--ldap-setup-admin-name', default=None, help="Ambari username for LDAP setup", dest="ldap_setup_admin_name")
+  parser.add_option('--ldap-setup-admin-password', default=None, help="Ambari password for LDAP setup", dest="ldap_setup_admin_password")
```

Comment:
Trying to be more generic about this, maybe rename to 'ambari-admin-password'.

This may come in handy in the future when other sets of properties are moved to the Ambari DB.